.github/workflows/kind.yaml

```diff
@@ -26,11 +26,12 @@ jobs:
           make build
           IMG=flotta-operator:latest make docker-build
           kind load docker-image flotta-operator:latest
-          timeout 1m make deploy IMG=flotta-operator || retVal=$?
+          IMG=flotta-operator TARGET=kind make deploy || retVal=$?
```

Comment:
@machacekondra do you think we need to have a timeout for this? in that case it will have to be 2m (the originally 1m + 1m for the cert-manager deployment from within the `make deploy`)

Reply:
Yes, I think we should keep it, and lets just increase it, as you suggest.